kind/actions/cluster_actions.go

```diff
@@ -149,13 +150,23 @@ func KubeadmInit(clusterName string) error {
 	if err != nil {
 		return err
 	}
+	// Upload certs flag changed to non-experimental in >= 1.15
+	uploadCertsFlag := "--experimental-upload-certs"
+	parts := strings.Split(version, ".")
```

Comment:
probably better to use ParseSemantic from api-machinery/utils/version here.

Reply:
i was looking around for that lib. I knew it existed somewhere. I will use that, thank you!